redis/src/lib.rs

```diff
@@ -51,13 +51,13 @@ impl RedisPool {
 /// A `bb8::ManageConnection` for `redis::async::Connection`s.
 #[derive(Clone, Debug)]
 pub struct RedisConnectionManager {
-    client: Client,
+    connection_info: ConnectionInfo,
 }
 
 impl RedisConnectionManager {
     /// Create a new `RedisConnectionManager`.
-    pub fn new(client: Client) -> Result<RedisConnectionManager, RedisError> {
-        Ok(RedisConnectionManager { client })
+    pub fn new<T: IntoConnectionInfo>(info: T) -> Result<RedisConnectionManager, RedisError> {
```

Comment:
I think we should change `new()` to build the `Client` from `T: IntoConnectionInfo` here, so that we don't have to build a `Client` for every `connect()` call.